vic/drivers/image/src/vic_force.c

```diff
@@ -334,13 +334,12 @@ vic_force(void)
             atmos[i].air_temp[j] -= CONST_TKFRZ;
             // precipitation in mm/period
             atmos[i].prec[j] *= global_param.snow_dt;
-            // pressure in kPa
-            atmos[i].pressure[j] /= PA_PER_KPA;
```

Comment:
@wietsefranssen : currently the input units for classic driver and image driver are different - for example, for air pressure, the classic driver input is in kPa, and the image driver input is in Pa. As for output - the classic driver output pressure should be in kPa; I haven't looked in detail at the image driver output units yet, and it is possible that there might be a bug when outputing data. We are looking at the output part of the image driver soon, and will pin you if we make any changes related to the output units.